presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -166,4 +168,18 @@ public boolean isParquetDereferencePushdownEnabled()
     {
         return parquetDereferencePushdownEnabled;
     }
+
+    @NotNull
+    public WorkerType getWorkerType()
+    {
+        return workerType;
+    }
+
+    @Config("iceberg.execution.worker.type")
```

Comment:
To add some details, in this PR changes for the filter pushdown feature may not work as it is if we are using Java workers since there are no changes made in the Iceberg connector Worker code on the Java side as per the filter pushdown feature if required.

Reply:
Yes, and we need to make sure this pushdown-filter-enabled is set to false by default